code/game/objects/items/weapons/tools.dm

```diff
@@ -151,10 +152,9 @@
 	create_reagents(max_fuel)
 	reagents.add_reagent("fuel", max_fuel)
 	update_icon()
-	return
 
 /obj/item/weapon/weldingtool/examine(mob/user)
-	if(..(user, 0))
+	if(..(user, 1))

```

Comment:
I'm pretty sure this increases the radius of examination from same-tile to adjacent-tile, but that's no biggy. Was it allowing examination from elsewhere before?